paper-tab.html

```diff
@@ -169,8 +164,25 @@
 
           anchor.click();
         }
-      }
+      },
+
+      _createRipple: function() {
+        var ripple = Polymer.PaperInkyFocusBehaviorImpl._createRipple();
+        ripple.removeAttribute('center');
+        ripple.classList.remove('circle');
+        return ripple;
+      },
 
+      _asyncClick: function() {
```

Comment:
Yeah, it was collapsed after I pushed something over it and it decided this was a different line. This is overriding a function in IronButtonState.